macros/plot_vfat_and_channel_Scurve.py

```diff
@@ -1,4 +1,4 @@
-from plot_scurve import *
+from macros.plot_scurve import *
```

Comment:
Yeah not really sure how @cbravo135 intended the tools in /macros to be used but I can have a look and try to merge appropriately.